src/main/java/com/parasoft/demoapp/config/activemq/ActiveMQConfig.java

```diff
@@ -50,6 +50,21 @@ public class ActiveMQConfig {
     @Value("${spring.activemq.stomp-transport-connector-url}")
     private String stompUrl;
 
+    @Value("${spring.activemq.broker-url}")
+    private String brokerUrl;
+
+    @Value("${spring.activemq.user}")
+    private String username;
+
+    @Value("${spring.activemq.password}")
+    private String password;
+
+    @Value("org.apache.activemq.jndi.ActiveMQInitialContextFactory")
```

Comment:
No need to use `@Value("org.apache.activemq.jndi.ActiveMQInitialContextFactory")`, just use `private String initialContextClass = "org.apache.activemq.jndi.ActiveMQInitialContextFactory";`, the property connectionFactory as well.